crates/libtiny_logger/src/lib.rs

```diff
@@ -445,6 +445,20 @@ impl LoggerInner {
             MsgTarget::CurrentTab => {
                 // Probably a cmd error; these are ignored
             }
+
+            MsgTarget::Misc { name: MENTIONS_TAB } => {
+                // create mentions tab log file like we do with user tabs
+                let mut path = self.log_dir.clone();
+                path.push("mentions.txt");
+                if let Some(mut fd) = try_open_log_file(&path, &*self.report_err) {
+                    report_io_err!(self.report_err, print_header(&mut fd));
+                    f(&mut fd, &*self.report_err);
+                }
+            }
+
+            MsgTarget::Misc { .. } => {
```

Comment:
If you add an enum as suggested above, then we can explicitly match for unhandled cases here and make the code future-proof. If we later add another enum variant this code becomes a compile error so we don't forget to update it.